### PR TITLE
Issue 10

### LINKED ISSUE #10: Firebase Authentication using Email and Password
Hey!! I would like to work on adding login and sign-in functionality by using firebase authentication using email and password under gssoc'21. This issue is not yet present here. Thank You.

### PR DESCRIPTION
**Pull request by Abhishek-dab**
fixed #10

### CODE REVIEW

File: lib/login_screen.dart [hunk 1]
```diff
@@ -1,151 +1,240 @@
...
+                    FlatButton(
+                      onPressed: () async {
+                        if (_formkey.currentState.validate()) {
+                          setState(() => load = true);
+                          dynamic result =
+                              await _auth.signInWithEmailAndPassword(
+                                  _emailController.text,
+                                  _passwordController.text);
+
+                          if (result == null) {
```
Comment (infiniteoverflow) [inline lib/login_screen.dart:140]: Kindly add a else part , and navigate to the homescreen if authentication is successful

File: lib/loading.dart [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_spinkit/flutter_spinkit.dart';
+
+class Loading extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
```
Comment (infiniteoverflow) [inline lib/loading.dart:7]: Can you change the color of the loading Dots from red to white.

**Review by infiniteoverflow — CHANGES_REQUESTED** [review]

**Comment by Abhishek-dab** [comment]
@infiniteoverflow I added the changes which were requested.

**Comment by pratyushmp** [comment]
Good Job @Abhishek-dab !

**Comment by Abhishek-dab** [comment]
Thank you for merging the PR 😄.

**Pull request merged**